README.md

```diff
@@ -42,20 +42,29 @@ Use this [Copier](https://copier.readthedocs.io) template to generate an empty P
 
 ## How to use
 
-### Step 1/3: Install `copier`
+There are multiple ways to use this template:
 
+1. Creating a new package based on the template
+2. Applying the template's best practices to some pre-existing code
+3. Updating a package made with the template based on the latest template updates
+
+In all three cases, you will need to install copier first:
 ```shell
 pipx install copier
```

Comment:
Does user also need to install pipx?

Reply:
I've added instructions and a note about the difference between using `pip` or `pipx`